test/unit/tracks/text-tracks.test.js

```diff
@@ -58,18 +58,7 @@ QUnit.test('Player track methods call the tech', function(assert) {
 
 QUnit.test('TextTrackDisplay initializes tracks on player ready', function(assert) {
   let calls = 0;
-  const ttd = new TextTrackDisplay({
-    on() {},
-    addTextTracks() {
-      calls--;
-    },
-    getChild() {
-      calls--;
-    },
-    ready() {
-      calls++;
-    }
-  }, {});
+  const ttd = new TextTrackDisplay(TestHelpers.makePlayer(), {}, calls++);
```

Comment:
By making this change, the test actually becomes a no-op since `calls` never updates as needed via `ready()`, `getChild()` and `addTextTracks()`, instead, it will *always* get incremented here because it isn't wrapped in a function call.
If there's just one new method necessary, we should stub it in the object passed to TextTrackDisplay, otherwise, after making the player, we should replace the methods with new methods that increment and decrement `calls`.